mlir/include/mlir/Analysis/Presburger/Barvinok.h

```diff
@@ -99,6 +99,12 @@ QuasiPolynomial getCoefficientInRationalFunction(unsigned power,
                                                  ArrayRef<QuasiPolynomial> num,
                                                  ArrayRef<Fraction> den);
 
+/// Find the number of terms in a generating function, which is
+/// a quasipolynomial in the parameter space of the input function.
+/// The generating function must be such that for all values of the
+/// parameters, the number of terms is finite; else it returns 0.
```

Comment:
did you prove it returns zero otherwise? if not, remove the part after the semicolon